drivers/mtd_spi_nor/mtd_spi_nor.c

```diff
@@ -44,15 +44,27 @@
 #define TRACE(...)
 #endif
 
-#ifndef MTD_SPI_NOR_WRITE_WAIT_US
-#define MTD_SPI_NOR_WRITE_WAIT_US (50 * US_PER_MS)
-#endif
-
 #define MTD_32K             (32768ul)
 #define MTD_32K_ADDR_MASK   (0x7FFF)
 #define MTD_4K              (4096ul)
 #define MTD_4K_ADDR_MASK    (0xFFF)
 
+#ifndef MTD_SPI_NOR_WAIT_C_ER
+#define MTD_SPI_NOR_WAIT_C_ER       (16 * US_PER_SEC)
+#endif
+
+#ifndef MTD_SPI_NOR_WAIT_S_ER
+#define MTD_SPI_NOR_WAIT_S_ER       (40 * US_PER_MS)
+#endif
+
+#ifndef MTD_SPI_NOR_WAIT_32K_ER
+#define MTD_SPI_NOR_WAIT_32K_ER     (20 * US_PER_MS)
+#endif
+
+#ifndef MTD_SPI_NOR_WAIT_4K_ER
+#define MTD_SPI_NOR_WAIT_4K_ER      (10 * US_PER_MS)
+#endif
```

Comment:
Are all these numbers based on a standardized spec?

Reply:
No, each flash might have different timings here. It might need some fine-tuning for your flash.

Reply:
> No, each flash might have different timings here. It might need some fine-tuning for your flash.

Does it make sense to move them to the flash params if they are model-specific?